src/styles/blocks/navigation.scss

```diff
@@ -0,0 +1,45 @@
+.nav {
+  &__list {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    list-style-type: none;
+    align-items: end;
+  }
+
+  &__item {
+    text-transform: uppercase;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    display: block;
+    height: 60px;
+    line-height: 60px;
+    text-decoration: none;
+    color: $nav-link-color;
+
+    &:hover {
+      color: $nav-link-hover-color;
+    }
+  }
+}
+
+.is-active {
+  color: $nav-link-hover-color;
+  position: relative;
+}
+
+.is-active::after {
```

Comment:
The pseudo-element `::after` for `.is-active` should also be nested inside its block to maintain the BEM structure and styling encapsulation.